articles/azure-functions/durable/durable-functions-singletons.md

```diff
@@ -157,15 +157,15 @@ async def main(req: func.HttpRequest, starter: str) -> func.HttpResponse:
 
     existing_instance = await client.get_status(instance_id)
 
-    if existing_instance != None or existing_instance.runtime_status in ["Completed", "Failed", "Terminated"]:
+    if not existing_instance or existing_instance.runtime_status not in [df.OrchestrationRuntimeStatus.Pending, df.OrchestrationRuntimeStatus.Running, None]:
```

Comment:
Thank you for this contribution! Indeed, the type of `runtime_status` became an `Enum` after we originally made these docs, thanks for catching this!

**I have one concern though**: The samples for JS and C# all check against `Completed`, `Failed`, and `Terminated`. I realize you're doing a kind of contrapositive check here, which is equivalent, but it turns Python's example into the "odd one out". As a result, while I realize your change might be easier to read, I would prefer to keep this sample doing the same checks as in the other PLs. This keeps them consistent :)

I suggest the following change:

```suggestion
    if existing_instance is None or existing_instance.runtime_status in [df.OrchestrationRuntimeStatus.Completed, df.OrchestrationRuntimeStatus.Failed, df.OrchestrationRuntimeStatus.Terminated]:
```